bundle/edu.gemini.spdb.reports.collection/src/main/scala/edu/gemini/dbTools/ephemeris/TcsEphemerisExport.scala

```diff
@@ -169,11 +168,19 @@ object TcsEphemerisExport {
           p  <- files.write(hid, em)
         } yield p
 
-      def lookupEphemeris(hid: HorizonsDesignation): TryExport[EphemerisMap] =
-        HorizonsService2.lookupEphemerisE[EphemerisElement](hid, site, new Date(start.toEpochMilli), new Date(end.toEpochMilli), ElementCount) {
+      def lookupEphemeris(hid: HorizonsDesignation): TryExport[EphemerisMap] = {
+        val s = start.toEpochMilli
+        val e = end.toEpochMilli
+        HorizonsService2.lookupEphemerisE[EphemerisElement](hid, site, new Date(s), new Date(e), ElementCount) {
```

Comment:
`Instant` might be the best choice here, I'm not sure, but the underlying horizons service is expecting `Date`s.  For anything new I stick with the new Java time classes.